web/ui/static/js/graph/index.js

```diff
@@ -711,13 +711,11 @@ Prometheus.Graph.prototype.updateGraph = function() {
     // When stacked is toggled
     var max = 0;
     self.data.forEach(function(timeSeries) {
-      var currSeriesMax = 0;
       timeSeries.data.forEach(function(dataPoint) {
-        if (dataPoint.y > currSeriesMax && dataPoint.y != null) {
-          currSeriesMax = dataPoint.y;
+        if (dataPoint.y > max && dataPoint.y != null) {
```

Comment:
Hmm I don't think it is correct... If you have 2 timeseries with values `[t0=1, t1=2, t3=3]` and `[t1=2, t2=2, t3=2]`, the computed max value would be `3` while it should be `5`.